.travis.yml

```diff
@@ -1,18 +1,26 @@
-language: php
-
 sudo: false
 
+language: php
+
 cache:
-  directory:
+  directories:
     - $HOME/.composer/cache
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7
-  - hhvm
+    - 7
+    - hhvm
+
+install:
+    - composer self-update
+    - composer --prefer-source install
+
+script:
+    - ./vendor/bin/phpcs -p --extensions=php --standard=ruleset.xml src
+    - ./vendor/bin/phpunit -v --coverage-clover ./build/logs/clover.xml
+
+after_script:
+    - ./vendor/bin/coveralls -v
 
-before_script:
-    - composer install -n
+matrix:
+    allow_failures:
+        - php: hhvm
```

Comment:
you should just remove HHVM entirely for now, as it cannot work: they provide a PHP 5.6 runtime, not a PHP 7 one currently, meaning they cannot install deps